html-syntax/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+
+<html>
+
+  <img src="https://assets.pokemon.com/assets/cms2/img/pokedex/full/890.png" alt="A pokemon." title="A mosntrous pokemon.">
```

Comment:
make sure that all tags which you want to display on the screen are within the body tags

Reply:
![image](https://user-images.githubusercontent.com/96906988/151098052-d261c8db-65a5-4042-8cb9-f14c21da3e71.png)